ext/ext_skel.php

```diff
@@ -31,6 +31,50 @@ function error($message) {
 /* {{{ print_help
  */
 function print_help() {
+    echo <<< HELP
+WHAT IT IS
+
+  It's a tool for automatically creating the basic framework for a PHP extension.
+
+HOW TO USE IT
+
+  Very simple. First, change to the ext/ directory of the PHP sources. Then
+  run the following
+
+    php ext_skel.php --ext extension_name
+
+  and everything you need will be placed in directory ext/extension_name.
+
+  If you don't need to test the existence of any external header files,
+  libraries or functions in them, the extension is ready to be compiled in
+  PHP. To compile the extension, run the following:
+
+    ./buildconf; ./configure --enable-extension_name; make
```

Comment:
I think this should do the same as what `print_success()` does below in regards to `make` vs `nmake` and the `./` prefix, else it looks good.